snippets/app-router/catch-all.mdx

```diff
@@ -16,7 +16,7 @@ export async function generateStaticParams() {
 }
 
 export default async function Page({ params }: { params: ParsedUrlQuery }) {
```

Comment:
The `ParsedUrlQuery` type is now outdated. We should probably just inline it here instead of defining it as a type

Reply:
Good catch! This what you were thinking?
![CleanShot 2024-12-18 at 10 31 16@2x](https://github.com/user-attachments/assets/308b01cc-9b79-42ac-b29d-5b92a063113f)

Reply:
Yeah, that's perfect